### PR TITLE
sample scripts updated

### PR DESCRIPTION
**Pull request by fuzious**
- examples updated

### CODE REVIEW

**Review by ujjwalgupta94 — APPROVED** [review]
looks good

File: examples/POS-client/ERC1155/burnSingle.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ const maticPOSClient = utils.getMaticPOSClient()
 
 const execute = async () => {
   try {
-    const tx = await maticPOSClient.burnSingleERC1155(config.child.DERC1155, config.user.tokenId, config.user.amount)
-    console.log(tx.transactionHash) // eslint-disable-line
+    const tx = await maticPOSClient.isSingleERC1155ExitProcessed(
```
Comment (ujjwalgupta94) [inline examples/POS-client/ERC1155/burnSingle.js:7]: previous example was for `burnSingleERC1155` ? why we made to `isSingleERC1155ExitProcessed `

File: examples/POS-client/ERC1155/depositSingle.js [hunk 2]
```diff
@@ -7,8 +7,14 @@ const execute = async () => {
     const tx = await maticPOSClient.depositSingleERC1155ForUser(
       config.root.DERC1155,
       config.user.address,
-      config.user.tokenId,
-      config.user.amount
+      '123',
```
Comment (ujjwalgupta94) [inline examples/POS-client/ERC1155/depositSingle.js:10]: can we comment besdiee hardcoded value, as what they represents . e.g - what is '123'

File: examples/POS-client/ERC721/approve.js [hunk 3]
```diff
@@ -4,11 +4,14 @@ const maticPOSClient = utils.getMaticPOSClient()
 
 const execute = async () => {
   try {
-    const tx = await maticPOSClient.approveERC721ForDeposit(config.root.DERC721, config.user.tokenId)
+    const tx = await maticPOSClient.approveERC20TokensForDeposit(config.root.DERC20, config.user.amount, {
```
Comment (ujjwalgupta94) [inline examples/POS-client/ERC721/approve.js:7]: the example was for ERC721 but you have changed it to ERC20

**Review by ujjwalgupta94 — CHANGES_REQUESTED** [review]

File: examples/POS-client/ERC1155/depositSingle.js (hunk 2 above)
Comment (fuzious) [reply examples/POS-client/ERC1155/depositSingle.js]: done

File: examples/POS-client/ERC1155/burnSingle.js (hunk 1 above)
Comment (fuzious) [reply examples/POS-client/ERC1155/burnSingle.js]: don't know how the files got rearranged. fixed

**Review by ujjwalgupta94 — APPROVED** [review]
looks good

**Pull request merged**